project/profiles/capacity4more/themes/c4m/kapablo/templates/page.tpl.php

```diff
@@ -93,103 +93,109 @@
     </section>
   <?php endif; ?>
 
-  <!--  TOP MENU -->
```

Comment:
@ordavidil
We have group and project content types, that require different page templates.
However, they're both OG group (node).
The difference between them is their $og_group->type.
Couldn't find a way to assign different templates to differentiate between the 2.